<?xml version="1.0"?>
<launch>

	<param name="robot_description" command="$(find xacro)/xacro.py '$(find youbot_description)/robots/youbot_arm_only.urdf.xacro'"/>

     <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
        <param name="use_gui" value="true"/>
    </node>

	<node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher"/>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find youbot_arm_control)/rviz/youbot_arm_model.rviz"/>

    <node pkg="tf" type="static_transform_publisher" name="ground_link" args="-0.024 0.0 0.032 0.0 0.0 0.0 ground_link base_link 100"/>
</launch>
